index.html

```diff
@@ -9,18 +9,27 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <p>“Hello World, we are software developers!”</p>
-    <!--
-
-    Hello Camper!
-
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
-    - The freeCodeCamp Team
-
-    -->
+    <header id="header">
+      <img id="header-img" src="https://imgs.search.brave.com/1koirRHXnoNOUT7tu4uRCjySlKC1oaFuJD5-0hoeTzs/rs:fit:780:438:1/g:ce/aHR0cHM6Ly9pbWcx/LmdydW5nZS5jb20v/aW1nL3VwbG9hZHMv/MjAxNy8wNC9oaWRk/ZW4tbWVhbmluZy1i/ZWhpbmQtZmFtb3Vz/LWxvZ29zLTc4MHg0/MzhfcmV2MS5qcGc" alt="logo" />
+      <nav id='nav-bar'>
+        <a class='nav-link' href='#home'>Home</a>
+        <a class='nav-link' href='#about'>About</a>
+        <a class='nav-link' href='#product'>Product</a>
+      </nav>
+    </header>
+    <section id='home'>
+      This is the home section
+    </section>
+    <section id='about'>
+      This is the about section
+    </section>
```

Comment:
Consider giving your text a  class name, and a tag name such as paragraph tags or other tags. for readability, easy styling, and maintenance.